### DISCUSSION TITLE
Will there be a Kinetic Kudu release?

### DISCUSSION THREAD

**Discussion by pmahnke** [regolith-linux/regolith-desktop#756] [General]
I don't think I have seen any information on the release plan for Ubuntu 22.10; however, I might have missed it in the move from slack to github discussions.

**Comment by kgilmer** [comment]
Hi @pmahnke , there hasn't been much mention of it that I'm aware of but my plan is that Regolith 2.2 will support Ubuntu 22.10.  I am not committed yet to an ISO release.  I've added a task to the 2.2 project for tracking, here: https://github.com/orgs/regolith-linux/projects/25

**Comment by kgilmer** [comment]
![Screenshot from 2022-10-22 16-40-45](https://user-images.githubusercontent.com/49683/197366331-d71379c4-27ec-4945-9457-297f4d77a14b.png)

A testing build of Regolith 2.2 for 22.10 is available in the `unstable` stage.  This is the `apt` line:

```
https://regolith-desktop.org/unstable-ubuntu-kinetic-amd64 kinetic main
```

I've done some smoke testing, and haven't found any issues yet.

**Reply by pmahnke** [reply]
Just to mention, it seems to be all working fine for me as well.

**Reply by rachmadaniHaryono** [reply]
thank you for the guide

> add apt line

basically instead of 

```bash
echo deb "[arch=amd64 signed-by=/usr/share/keyrings/regolith-archive-keyring.gpg] \
https://regolith-desktop.org/release-ubuntu-jammy-amd64 jammy main" | \
sudo tee /etc/apt/sources.list.d/regolith.list
```

run this

```bash
echo deb "[arch=amd64 signed-by=/usr/share/keyrings/regolith-archive-keyring.gpg] \
https://regolith-desktop.org/unstable-ubuntu-kinetic-amd64 kinetic main" | \
sudo tee /etc/apt/sources.list.d/regolith.list
```

from this https://regolith-desktop.com/

**Reply by rachmadaniHaryono** [reply]
before this there is bug that time indicator is missing from the tray. after adding the unstable ppa and reinstall i3xrocks-time time indicator appear again
